Readme.md

```diff
@@ -0,0 +1,65 @@
+# HTML FORMS
+
+> Creating a Smashing Magazine Website
+
+![screenshot](images/screenshot.PNG)
```

Comment:
- Improve your Readme file. It should be descriptive of the project. You are using the correct template but now you need to fill it with information about your project. You can add details about what you did in this project, why you did it, how you did it...